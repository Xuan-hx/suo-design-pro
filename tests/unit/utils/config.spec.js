import { api } from './config'

describe('配置文件测试', () => {
  it('测试配置文件是否变动', () => {
    expect(api).toMatchSnapshot({
      develop: 'http://xxxx:8080',
      mock: 'http://xxxx',
      feature: 'http://xxxx',
      test: 'http://xxxx',
      production: 'http://xxxx'
    })
  })
})
