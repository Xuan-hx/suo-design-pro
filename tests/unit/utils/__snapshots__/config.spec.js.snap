// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`配置文件测试 测试配置文件是否变动 1`] = `
Object {
  "develop": "http://xxxx:8080",
  "feature": "http://xxxx",
  "mock": "http://xxxx",
  "production": "http://xxxx",
  "test": "http://xxxx",
}
`;
